packages/block-editor/src/components/block-patterns-list/index.js

```diff
@@ -43,18 +38,34 @@ function BlockPattern( { isDraggable, pattern, onClick } ) {
 					onDragStart={ onDragStart }
 					onDragEnd={ onDragEnd }
 				>
-					<BlockPreview
-						blocks={ blocks }
-						viewportWidth={ viewportWidth }
-					/>
-					<div className="block-editor-block-patterns-list__item-title">
-						{ pattern.title }
-					</div>
-					{ !! pattern.description && (
-						<VisuallyHidden id={ descriptionId }>
-							{ pattern.description }
-						</VisuallyHidden>
-					) }
+					<CompositeItem
+						role="option"
+						as={ 'div' }
+						{ ...composite }
+						className="block-editor-block-patterns-list__item"
+						onClick={ () => onClick( pattern, blocks ) }
+						onKeyDown={ ( event ) => {
+							if (
+								ENTER === event.keyCode ||
+								SPACE === event.keyCode
+							) {
+								onClick( pattern, blocks );
+							}
+						} }
```

Comment:
I don't think this `onKeyDown` is needed anymore as `CompositeItem` already handles this.
```suggestion
```